CHANGELOG.md

```diff
@@ -7,6 +7,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 ## [master]
 
 ### Added
+
+- [web] Added getting REMOTE_USER header
```

Comment:
Maybe: "Support of the REMOTE_USER header to enable external authentication system" ?